Source/EasyNetQ/Extensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace EasyNetQ
+{
+    public static class Extensions
+    {
+        public static void SafeDispose(this IDisposable disposable)
+        {
+            try
+            {
+                disposable.Dispose();
```

Comment:
[now](https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/121) `ModelBase.Dispose()` will not throw exception anymore, but we should leave this `SafeDispose` to be sure will never happen again that calling dispose on EasyNetQ throws exception, that could cause windows services to stuck indefinitely.